client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -32,10 +37,17 @@ export const InboundReturnEditModal = ({
   outboundShipmentLineIds,
   customerId,
   onClose,
+  modalMode,
 }: InboundReturnEditModalProps) => {
-  const t = useTranslation('distribution');
+  const t = useTranslation(['distribution', 'replenishment']);
```

Comment:
Hm, this probably means the translation strings here should be in `common`, rather than pulling the replenishment translations into distribution?